cfgov/v1/templatetags/mega_menu.py

```diff
@@ -0,0 +1,14 @@
+from wagtail.wagtailcore import hooks
+
+from v1.models.menu_item import MenuItem
+
+
+@hooks.register('before_serve_shared_page')
```

Comment:
Per [the Wagtail docs](http://docs.wagtail.io/en/v1.13.1/reference/hooks.html) should this `register` command and this function live in `cfgov/v1/wagtail_hooks.py`? There's no guarantee this file gets imported at runtime. I know it's nice to have this next to the function that checks this attribute; maybe a comment in `get_menu_items` could refer here?

Reply:
Thanks, @chosak -- this should be fixed in https://github.com/cfpb/cfgov-refresh/pull/3396/commits/53ab1541cfdcb3558d4335eecac456d47af16091.